### ISSUE TITLE
Make train.py compatible with the data_loader and data_processor classes

### ISSUE THREAD

**Issue by aribrill** [bryankim96/ctalearn#17]
Split data_processing_settings into data_loading_settings and data_processing_settings. There should be four dictionaries of data-related settings:
data_input_settings: for TensorFlow Estimator input_fn
data_loading_settings: for loading HDF5 data (methods directly implemented in load_HDF5_data.py)
Includes validation_split, min_num_tels and cut_condition, use_telescope_position, chosen_telescope_types, model_type
data_processing_settings: for processing data in process_data.py
Includes sort_telescopes_by_trigger, crop_images, log_normalize_charge, all cleaning and cropping options
image_mapping_settings: for mapping pixels to arrays in image_mapping.py, to be implemented in a separate issue #10 

Rewrite the section stating on line 140 with "if data_format == 'hdf5':" to use the data_loader class methods.

Depends on #15 and #16.

**Comment by aribrill** [comment]
Closed by #29
